src/player.py

```diff
@@ -1,2 +1,10 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+class Player:
+  def __init__(self, name, current_room):
+    self.name = name
+    self.current_room = current_room
+
+  def move(self, room):
+    if room:
+      self.current_room = self.current_room.new_room(room)
```

Comment:
I like how you're linking the player to the room in this manner. I did all this logic in the game loop, but as long as this works, it's correct.